x/vaults/keeper/vault.go

```diff
@@ -152,27 +155,24 @@ func (k *Keeper) MintCoin(
 	sender sdk.AccAddress,
 	mint sdk.Coin,
 ) error {
-	params := k.GetParams(ctx)
-	if mint.Denom != params.MintDenom {
-		return fmt.Errorf("minted denom must be %s", types.DefaultMintDenom)
-	}
 	vault, err := k.GetVault(ctx, vaultId)
 	if err != nil {
 		return err
 	}
-	if sender.String() != vault.Owner {
-		return fmt.Errorf("sender is not the vault owner")
-	}
```

Comment:
why remove this check?

Reply:
This could lead to someone else minting the money to a point where it could be liquidated without the owner knowing.